<?xml version="1.0" ?>
<launch>
  <!-- params -->
  <param name="forest_file" value="$(find leg_follower)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/scan" />
  <param name="fixed_frame" value="/base_footprint"/>
  <param name="velocity_topic" value="/cmd_vel"/>
  <param name="scan_frequency" value="10"/>
  <!-- the launch file of your robot driver -->
  <include file="$(find autolabor_pro1_driver)/launch/driver.launch"/>
  <!-- the launch file of your lidar driver -->
  <include file="$(find autolabor_pro1_driver)/launch/urg04lx_lidar.launch"/>
   <node pkg="tf" type="static_transform_publisher" name="laser_base" args="0.13 0 0 0 0 0 base_footprint laser 50" />
  <!-- run detect_leg_clusters -->  
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run joint_leg_follower -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_follower" output="screen"/>
 
  <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/>
  <!-- make sure you have give the execute permission to this python file -->
  <node pkg="leg_tracker" type="people_tracker.py" name="track_people" output="screen"/>
</launch>

